### PR TITLE
Plehanova Anastasia eh1 t1 homework done

### PR DESCRIPTION
**Pull request by Littledrop777**

### CODE REVIEW

**Review by Alex99905 — APPROVED** [review]

**Review by Svetlana101010 — APPROVED** [review]

**Review by Svetlana101010 — APPROVED** [review]

**Review by Svetlana101010 — DISMISSED** [review]

File: homework/english/plehanova_Anastasia/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Упражнения Present Simple и Present Continuous
+
+### Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+
+1. Is your mum working today
+2. Why are this boys to talking now?
+3. Where does he live?
+4. Their mother cooks very well?
```
Comment (VolhaZh) [inline homework/english/plehanova_Anastasia/lecture1/task1.md:8]: does their ...

File: homework/english/plehanova_Anastasia/lecture1/task1.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+12. Look! The boys are comming home.
+13. Does Maila like Chinese food?
+14. My grandmother doesn't work. She is on pension.
+15. What is your brother watching now?
+16. Where do you live?
+17. Do you play the piano well?
+18. Where is their brother swimming now?
+19. What do they eat at breakfast?
+20. Are they swimming at the moment?
+21. Mike doesn't wear a tie today.
```
Comment (VolhaZh) [inline homework/english/plehanova_Anastasia/lecture1/task1.md:25]: today - marker word - present continious  needed

**Review by VolhaZh — CHANGES_REQUESTED** [review]

**Review by Alex99905 — DISMISSED** [review]
Lgfm

File: homework/english/plehanova_Anastasia/lecture1/task1.md (hunk 1 above)
Comment (Littledrop777) [reply homework/english/plehanova_Anastasia/lecture1/task1.md]: change

File: homework/english/plehanova_Anastasia/lecture1/task1.md (hunk 2 above)
Comment (Littledrop777) [reply homework/english/plehanova_Anastasia/lecture1/task1.md]: change

**Review by VolhaZh — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]
Lgfm

**Review by tamenkessho — APPROVED** [review]
Well done

**Review by Alex99905 — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]

**Pull request merged**